### DISCUSSION TITLE
Is this project still maintained?

### DISCUSSION THREAD

**Discussion by havana7** [OrchardCMS/OrchardCore.Commerce#276] [General] [answered]
I would like to know if it is possible to use it as a project in production.

**Comment by sarahelsaig** [comment, marked as answer]
Hi, yes the project is maintained. At this moment we aren't working on new features, but we'll review any incoming pull requests and address any security issues. It can be used in production, we have a contact whose OrchardCore.Commerce powered site is in pre-production and will launch into production soon.
